crates/services/compression/src/ports/block_source.rs

```diff
@@ -17,7 +17,7 @@ impl BlockWithMetadataExt for BlockWithMetadata {
     }
 
     fn height(&self) -> &BlockHeight {
-        <Self as BlockWithMetadataExt>::block(&self)
+        <Self as BlockWithMetadataExt>::block(self)
```

Comment:
Do we still need to disambiguate here btw?

Reply:
a7dd928951